airbyte-cdk/python/unit_tests/sources/declarative/interpolation/test_jinja.py

```diff
@@ -57,24 +58,12 @@ def test_literals(s, value):
             "{{ stream_slice['stream_slice_key'] }}",
             "hello",
             id="test_get_value_from_stream_slice"),
-        pytest.param(
-            {},
-            "{{ stream_slice['stream_slice_key'] }}",
-            None,
-            id="test_get_value_from_stream_slice_no_slice"
-        ),
         pytest.param(
             {"stream_slice": {"stream_slice_key": "hello"}},
             "{{ stream_partition['stream_slice_key'] }}",
             "hello",
             id="test_get_value_from_stream_slicer"
         ),
-        pytest.param(
```

Comment:
this is not true anymore. we'll raise an exception if the variable is not present in the context